src/tdvp_generic.jl

```diff
@@ -118,7 +118,7 @@ function tdvp(solver, PH, t::Number, psi0::MPS; kwargs...)
     end
     isdone && break
   end
-  return psi
+  return psi, info
```

Comment:
I don't think we should return the `info` here, I think we should just pass it to the observer. Then users can store it for difference steps, choose when to save it, etc.

Reply:
I do not understand what you mean by this. The only ways I can think if right now to pass something on to the user are by
- returning the info from the function
- passing a mutable struct as an (optional) function parameter.

Are you suggesting I do the second? And if not, what then?

Reply:
done